modules/core/src/lib/deck.js

```diff
@@ -640,21 +656,11 @@ export default class Deck {
     // Needs to be done before drawing
     this._updateAnimationProps(animationProps);
 
-    // Check if we need to redraw
-    const redrawReason = this.needsRedraw({clearRedrawFlags: true});
-
+    // Perform picking request if any
     this._pickAndCallback();
 
-    if (!redrawReason) {
-      return;
-    }
-
-    this.stats.get('Redraw Count').incrementCount();
-    if (this.props._customRender) {
-      this.props._customRender();
-    } else {
-      this._drawLayers(redrawReason);
-    }
+    // Redraw if necessary
+    this.redraw(false);
```

Comment:
Just a nit: My concerns about boolean parameters are already well know so I was planning to keep my peace, but I couldn't help but notice that we already felt the need to add a comment explaining what the `false` parameter means, and this is not even in application code, but in the very same file as the function definition....